contracts/event/EventFactory.sol

```diff
@@ -54,7 +54,7 @@ contract EventFactory is NRC223Receiver {
         bytes memory funcHash = data.sliceBytes(0, 4);
         bytes memory params = data.sliceBytes(4, data.length - 4);
         bytes32 encodedFunc = keccak256(abi.encodePacked(funcHash));
-        if (encodedFunc == keccak256(abi.encodePacked(hex"2b2601bf"))) {
+        if (encodedFunc == keccak256(abi.encodePacked(hex"662edd20"))) {
```

Comment:
why the hash is changed here?

Reply:
cause adding the two new params changed it. didnt change it before, but this is the correct hash.